docker/server/entrypoint.sh

```diff
@@ -61,7 +64,40 @@ do
     fi
 done
 
+# if clickhouse user is defined - create it (user "default" already exists out of box)
+if [ -n "$CLICKHOUSE_USER" ] && [ "$CLICKHOUSE_USER" != "default" ]; then
```

Comment:
Or password for default user is not empty

Reply:
Updated